src/components/Layout/Header/NavBar/UserMenu.tsx

```diff
@@ -89,14 +91,38 @@ type WalletButtonProps = {
   onConnect: () => void
 } & Pick<InitialState, 'walletInfo'>
 
-const WalletButton = ({ isConnected, walletInfo, onConnect }: WalletButtonProps) => {
+const WalletButton: FC<WalletButtonProps> = ({ isConnected, walletInfo, onConnect }) => {
+  const [walletLabel, setWalletLabel] = useState('')
+  const bgColor = useColorModeValue('gray.300', 'gray.800')
+
+  useEffect(() => {
+    setWalletLabel('')
```

Comment:
Is this needed since you set it on all other branches, including that unhappy path early return on next line?

Reply:
oh yeah I just added this for testing and forgot to remove it